### ISSUE TITLE
REPL in the website ?

### ISSUE THREAD

**Issue by DjebbZ** [cljs/site#26]
The README says 

> REPL page using replumb, used by different projects

I'd really to see it happens in one form or another. I'd like to discuss about possible integrations and UIs. 

What do you think about :
- having a "Try it" button for each documented stuff, that preloads with a simple example and allows you to evaluate more forms ?
- dont' remember where I saw it, but I like the possibility of clicking an example and loading it in the REPL
- even if I like the idea of a dedicated page, I think for documentation what's nice is having a REPL for each form. Of course, it's even better to have both.

What do you think ?

**Comment by shaunlebron** [comment]
yeah I kinda wanted to try out klipse or some other REPL for each API details page.  The REPL stuff is currently too slow to load by default (because a bootstrapped eval environment cannot be optimized in advance), but I think I like your idea of loading it when "Try it" is clicked.  Will take a PR!

**Comment by DjebbZ** [comment]
Great ! Will try soon hopefully. Indeed this is the reason why I want it
behind a try button to load all stuff on demand.
